doc/source/whatsnew/v0.21.1.txt

```diff
@@ -74,6 +74,9 @@ Indexing
 I/O
 ^^^
 
+- Bug in `StataReader` not converting date/time columns with display formatting addressed (:issue:`17990`)
```

Comment:
can you do `` class:`~pandas.io.stata.StataReader` ``. Can you make a bit more clear what is being fixed here.